material_3_demo/test/color_screen_test.dart

```diff
@@ -1,4 +1,7 @@
-// ignore_for_file: avoid_types_on_closure_parameters
```

Comment:
You might want modify the analysis options config to drop this lint.

Reply:
Got it. Added it back:) Thanks!